modulus/experimental/resdiff/training/training_loop.py

```diff
@@ -195,11 +197,11 @@ def print0(self, *message):
         from userlib.auto_resume import AutoResume
         AutoResume.init()
     except ImportError:
-        print('AutoResume not imported')
+        raise ImportError('AutoResume not imported')
```

Comment:
Note the change in behavior: the old code will print the message and continues, the new one - re-raises the exception. Is this what you expect?